how-to/transfer-project-ownership.rst

```diff
@@ -0,0 +1,39 @@
+Transferring ownership of a Launchpad project
+=============================================
+
+Transferring the ownership of a project in Launchpad, takes more than simply
+changing the maintainer of a project when there is private content
+associated with it.
+
+This is particularly evident for private projects, when after updating the
+project's maintainer, the maintainer themselves might not be able to see the
+private project they maintain.
+
+This is because the permissions to see and update a project lie on its sharing
```

Comment:
Maybe `are set (or defined) via its sharing policies` instead of `lie on`?

Reply:
Updated